src/actors.rs

```diff
@@ -222,8 +218,16 @@ where
             futures::pin_mut!(tasks);
             futures::select! {
                 t = tasks => {
-                    if t? == 0 {
-                        smol::Timer::new(std::time::Duration::from_millis(3600)).await;
+                    match t {
+                        Err(coil::FetchError::Timeout) => {
+                            log::warn!("Timeout waiting for task to start");
+                            continue;
+                        },
+                        v => {
+                            if v? == 0 {
+                                smol::Timer::new(std::time::Duration::from_millis(3600)).await;
+                            }
+                        }
```

Comment:
Maybe we can do something like

```
Ok(v) if v == 0 => { … … },
Ok(_) => {} // but maybe we want to log something here?
Err(FailedLoadingJob) => { … … }
```

Not sure if it's better, but sometimes explicit is good.